build.gradle.kts

```diff
@@ -134,4 +134,13 @@ fun Project.configureMavenPublish() {
             }
         }
     }
+}
+
+apply(from = "docs.gradle.kts")
+
+tasks.register("deployDocsIfNeeded") {
+    val branch = System.getenv("TRAVIS_BRANCH")
+    if (branch == "master") {
+        dependsOn("deployDocs")
```

Comment:
Should we deploy docs on a PR build too ?
With this impl, it will

Reply:
Thanks, fixed.